lib/LDAPConnect.php

```diff
@@ -145,4 +148,29 @@ public function hasPasswordPolicy(): bool {
 		$ppDN = $this->ldapConfig->ldapDefaultPPolicyDN;
 		return !empty($ppDN);
 	}
+
+	/**
+	 * checks whether the LDAP server supports the passwd exop
+	 *
+	 * @param LDAP\Connection $connection LDAP connection to check
+	 * @return boolean either the user can or cannot
+	 */
+	public function hasPasswdExopSupport():bool {
```

Comment:
```suggestion
	public function hasPasswdExopSupport($connection):bool {
```
The parameter is missing?

Reply:
> Also this code will fail when there are 2 ldap configured and one of them has passwd support but not the other one. The cache should be by connection.

As far as I can see only one host can be configured for `LDAPConnect` so I think caching this per `LDAPConnect` is ok?

Reply:
I don’t know I’m lost.
@blizzz How does it work? Do the app event supports multiple LDAP configurations? This class seems to just take the first one from the list.

Reply:
Indeed only the first configured LDAP connection is being considered as of now.